test/workbox-build/node/entry-points/inject-manifest.js

```diff
@@ -0,0 +1,247 @@
+const expect = require('chai').expect;
+const path = require('path');
+const tempy = require('tempy');
```

Comment:
I'm just using this whenever we need to write to disk, with the assumption that the operating system knows how to clean up temporary files.

Reply:
I wasn't, with that devil-may-care attitude inspired by https://github.com/sindresorhus/tempy#why-doesnt-it-have-a-cleanup-method:

> Why doesn't it have a cleanup method?
>The operating system will clean up when needed. No point in us wasting resources and adding complexity.

I could add it a cleanup step, but the one thing that makes me hesitate is that all that AppVeyor flakiness seemed to be tied to failed attempts to clean up what could have been treated as temporary files